security/test_signing.py

```diff
@@ -88,7 +97,14 @@ def test_verify_signed_message(self, mock_get_public_keys):
 
 class ResponseSigningTests(TestCase):
     def setUp(self):
-        self.response = {
+        _payload = {"type_of_operation": 1, "originating_party": "TII"}
```

Comment:
Can we take this from env, instead of hardcode as it will be required in upstream base